documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -200,20 +201,39 @@ log.roll.ms=604800000
 # ...
 ----
 
-You can override these settings at the topic level using `segment.bytes` and `segment.ms`.
-Whether you need to lower or raise these values depends on the policy for segment deletion.
-A larger size means the active segment contains more messages and is rolled less often.
-Segments also become eligible for deletion less often.
+These settings can be overridden at the topic level using `segment.bytes` and `segment.ms`. 
+The choice to lower or raise these values depends on the policy for segment deletion. 
+A larger size means the active segment contains more messages and is rolled less often. 
+Segments also become eligible for deletion less frequently.
 
-You can set time-based or size-based log retention and cleanup policies so that logs are kept manageable.
-Depending on your requirements, you can use log retention configuration to delete old segments.
-If log retention policies are used, non-active log segments are removed when retention limits are reached.
-Deleting old segments bounds the storage space required for the log so you do not exceed disk capacity.
+In Kafka, log cleanup policies determine how older log data is managed.
```

Comment:
I guess with compact policy its not just the older data but also the current data?

```suggestion
In Kafka, log cleanup policies determine how log data is managed.
```